config/default.schema.json

```diff
@@ -1671,6 +1671,11 @@
 			"default": "false",
```

Comment:
is already set as true in dof app, but we should set it to true by default here too